Dockerfile

```diff
@@ -0,0 +1,178 @@
+FROM kappnav/base-runtime:8-jre-ubi-min-alt2 as liberty
```

Comment:
I'm sure this gets the job done, but wonder if there's a better way of layering the "inventory" container on top of the API container, not only to reduce (Dockerfile) code duplication but also to have a common base that shares the same footprint.

Reply:
I agree.  But this won't change until the kappnav API server can run as a separate deployment, which is backlog issue https://github.com/kappnav/issues/issues/5